### DISCUSSION TITLE
Welcome to pico-bteve Discussions!

### DISCUSSION THREAD

**Discussion by BRTSG-FOSS** [Bridgetek/pico-brteve#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our Pico-Eve community. We hope that you:
  * Ask questions you’re wondering about how to use Eve with Pico .
  * Share ideas on the applications to build with Pico-Eve.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by qombajn** [comment]
Hello! 
Could you tell me how to add bigger font to basic c example? I need one that takes 2/3 of the screen height.
